src/orquestra/quantum/openfermion/utils/operator_utils.py

```diff
@@ -138,6 +139,12 @@ def count_qubits(operator):
     Raises:
        TypeError: Operator of invalid type.
     """
+    # Handle PauliSum
+    if isinstance(operator, PauliSum):
```

Comment:
I think we won't need this anymore in "post openfermion" world.

Reply:
Same as above